components/Repositories/LotteryFactorChart.tsx

```diff
@@ -129,6 +134,20 @@ export default function LotteryFactorChart({
         </header>
       </section>
 
+      {yoloBannerOnClick && (
+        <button
+          onClick={yoloBannerOnClick}
+          className="flex items-center w-full shadow-sm !border !border-slate-300 px-0.5 py-0.5 gap-2 text-[0.7rem] rounded-full"
+        >
+          <div className="flex items-center font-medium gap-1 px-2 py-0.5 rounded-2xl bg-light-red-4 text-light-red-11">
+            <FaRegHandPeace />
+            YOLO Coders
+          </div>
+          Pushing commits directly to main
+          <FaArrowRight />
```

Comment:
can we add the classes `ml-auto mr-3` to push the arrow all the way to the right of the button?

Like so:
<img width="491" alt="image" src="https://github.com/open-sauced/app/assets/27322879/62e32d14-e22f-4214-a94d-eade14d7cd69">